plugin/plugins/filesystem/nodes/base.go

```diff
@@ -0,0 +1,45 @@
+package fsnodes
+
+import (
+	"context"
+
+	"github.com/djdv/p9/p9"
+	"github.com/djdv/p9/unimplfs"
+	logging "github.com/ipfs/go-log"
+	coreiface "github.com/ipfs/interface-go-ipfs-core"
+	corepath "github.com/ipfs/interface-go-ipfs-core/path"
+)
+
+const ( //device
+	dMemory = iota
+	dIPFS
```

Comment:
@Stebalien ^
Do we have a Go library that holds these values? I feel like that's worth having even if the table isn't expected to change rather than plugging them in per project.

(Our definitions are here https://github.com/ipfs/go-ipfs/pull/6612/files#diff-6c2c294fc60db12a68ce40457ff3ea5eR17)